scripts/src/lib/template/templates/gradle/kotlin/build.gradle.kts.eta

```diff
@@ -0,0 +1,113 @@
+plugins {
+	id("fabric-loom") version "1.0-SNAPSHOT"
+	id("maven-publish")<% if (it.kotlin) { %>
+	kotlin("jvm") version "<%= it.kotlin.kotlinVersion %>"<% } %>
+}
+
+java.sourceCompatibility = JavaVersion.<%= it.java.compatibility %>
+java.targetCompatibility = JavaVersion.<%= it.java.compatibility %>
+
+base.archivesName.set(project.properties["archives_base_name"] as String)
+version = project.properties["mod_version"] as String
+group = project.properties["maven_group"] as String
+
+repositories {
+	// Add repositories to retrieve artifacts from in here.
+	// You should only use this when depending on other mods because
+	// Loom adds the essential maven repositories to download Minecraft and libraries from automatically.
+	// See https://docs.gradle.org/current/userguide/declaring_repositories.html
+	// for more information about repositories.
+}
+<% if (it.dataGeneration || it.splitSources) { %>
+loom {
+<% if (it.splitSources) { %>	splitEnvironmentSourceSets()
+
+	mods {
+		create("modid") {
```

Comment:
Should the mod ID be a template here? I see it isn't that way in the groovy template either, but that feels wrong

Reply:
Im ok if this isnt done for this PR.

Reply:
true...